src/tastyscrapy/spiders.py

```diff
@@ -48,18 +49,46 @@ def on_login_response(self, response):
         if xjpath.path_lookup(payload, 'status') != ('success', True):
             raise LoginFailedException("Unable to log in using provided credentials.")
 
-        # emit an item which shows that we logged in successfully for audit tracking
-        yield LoginSuccess(created=datetime.utcnow().isoformat())
-
+        # lookup redirect url
         redirect, is_success = xjpath.path_lookup(payload, 'redirect')
 
+        # extract session key for cookie
+        session_key, session_value = xjpath.XJPath(payload)['session'].split('=')
+
         # start parsing items
         yield scrapy.Request(redirect if is_success else "https://del.icio.us/{}".format(
-            self.settings.get('DELICIOUS_USERNAME')), callback=self.on_bookmark_response)
+            self.settings.get('DELICIOUS_USERNAME')), callback=self.on_bookmark_response,
+            cookies = { session_key: session_value })
```

Comment:
We then take the session key and value and add them to the cookies dictionary when redirecting after login. After setting the `cookies` dictionary once, Scrapy will "remember" the cookie and send it with all future requests.